app/models/intake.rb

```diff
@@ -270,4 +270,16 @@ def get_or_create_spouse_auth_token
     update(spouse_auth_token: new_token)
     new_token
   end
+
+  def mixpanel_data
+    {
+      primary_filer_age_at_end_of_tax_year: primary_user&.age_end_of_tax_year.to_s,
+      spouse_age_at_end_of_tax_year: spouse&.age_end_of_tax_year.to_s,
+      primary_filer_disabled: had_disability,
+      spouse_disabled: spouse_had_disability,
+      intake_source: source,
+      intake_referrer: referrer,
+      intake_referrer_domain: referrer_domain,
+    }
```

Comment:
I want to piggy back on this to ask for a few more key metrics, if you don't mind:
- whether they have dependents
- whether they are filing joint
- whether they have any dependent under 6 (we already have a method for this I think)
- number of dependents (as a string, otherwise mixpanel will butcher the number)